lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -9,8 +9,8 @@ defmodule Kernel.SpecialForms do
   been automatically imported, in the same fashion as the functions and
   macros from the `Kernel` module.
 
-  These building blocks are defined in this module. Some of these special forms are lexical (like
-  `alias/2`, `case/2`, etc.). The macros `{}/1` and `<<>>/1` are also special
+  These building blocks are defined in this module. Some of these special forms are lexical (such as
+  `alias/2` and `case/2`). The macros `{}/1` and `<<>>/1` are also special
```

Comment:
I believe you want "or" here? 🤔

Reply:
I thouhgt of it. I think it is AND because both are lexical. See the following sentence, it also uses AND to convey a similar message.